library/sources/http-server/checkIfRequestIsBlocked.ts

```diff
@@ -48,6 +49,25 @@ export function checkIfRequestIsBlocked(
     return false;
   }
 
+  if (
+    context.remoteAddress &&
+    agent.getConfig().shouldOnlyAllowSomeIPAddresses() &&
```

Comment:
Do we need this check here "shouldOnlyAllowSomeIPAddresses", because we still want to block IPs on the blocklist

Reply:
Refactoring, but didn't had any effect on blocking